web-marketplace/src/legacy-pages/Dashboard/Dashboard.EditProfile.tsx

```diff
@@ -43,125 +54,292 @@ export const EditProfile = () => {
     profileBannerCardAtom,
   );
   const fileNamesToDeleteRef = useRef<string[]>([]);
+  const setProcessingModalAtom = useSetAtom(processingModalAtom);
+  const [error, setError] = useState<string | undefined>();
+  const {
+    selectedAccountAddress,
+    isOrganizationDashboard,
+    organizationDaoAddress,
+    organizationRbamAddress,
+    organizationProfile: organizationProfileFromContext,
+  } = useDashboardContext();
+  const updateOrganizationProfile = useUpdateOrganizationProfile();
+  const organizationProfile = organizationProfileFromContext ?? null;
```

Comment:
`organizationProfileFromContext` can already be potentially null so I don't think the logic line 67 is useful
```suggestion
  const {
    selectedAccountAddress,
    isOrganizationDashboard,
    organizationDaoAddress,
    organizationRbamAddress,
    organizationProfile,
  } = useDashboardContext();
  const updateOrganizationProfile = useUpdateOrganizationProfile();
```